### ISSUE TITLE
Users

### ISSUE THREAD

**Issue by serge-nikitin** [jscraft-org/mentoring-project-1#4]
Let's add users.
We need a table with fields: id, created_at, username, password.

We need such functionality: register, login.

Don't use passport.js, create your own middleware. 
You will need to use express-session module. You can find tutorials about this topic on Youtube, for example: https://www.youtube.com/watch?v=BnLOTQP5tZk

**Comment by marcpre** [comment]
I have created a simple login/logout system using `express-session`.

Within my `util/session.js` I created a `sessionChecker`, which I am using to check if a user is logged in or not.

Due to better readability I have created all my functions in `service/user` with `async-await`.

Still I am facing the following issue:

When calling my login function I want to check the username and password, however I get nothing back from the knex query.

The method I am referring to is the [login method](https://github.com/jscraft-org/mentoring-project-1/blob/master/service/user.js). Currently, I am not getting any result back from the res block.
```
        const res = await knex("users").select().where({
            username: username,
            password: pwdHash
        }).first()
```

I tried to rewrite it into a promise, however, I am getting the same output.

Any idea where the problem might be here located?

Appreciate your reply!
